shell/packages/sandstorm-db/db.js

```diff
@@ -415,9 +423,17 @@ ApiTokens = new Mongo.Collection("apiTokens");
 //       }
 //     }
 //     frontendRef :union {
-//        notificationHandle :Text;
-//        ipNetwork :Bool;
-//        ipInterface :Bool;
+//       notificationHandle :Text;
+//       ipNetwork :Bool;
+//       ipInterface :Bool;
+//       emailVerifier :group {
+//         services :List(String);
+//       }
+//       verifiedEmail :group {
+//         verifierId :Text;
+//         tabId :Text;
```

Comment:
If the app wants to persist a `VerifiedEmail`, it would need to store the `tabId` that the user had at the time it was requested. Remember that the point here is to ensure that the verification occurred in the `tabId` that the app thought it did. That said, I don't think apps would commonly store a `VerifiedEmail` long-term; once the verification handshake has completed, they may as well store the address itself.